chibuezeayogu/React-Math-Skills-Game
# #160871116  Implement how to play modal

**What does the PR do?**
- Adds `How To Play` Modal

**Description of Task to be completed?**
- explains how the game is played to the user 

**How should this be manually tested?**
- Clone the repository [here](https://github.com/chibuezeayogu/React-Math-Skills-Game.gitt)
-  cd into `React-Math-Skills-Game` and checkout to this branch `ft-add-how-to-play-160871116`
- run the command `npm run start:dev`
- run the command `npm test` to ensure all test are passing

**What are the relevant pivotal tracker stories?**
- [#160871116](https://www.pivotaltracker.com/story/show/160871116)

**Screenshots**
- N/A
